From feaf8a8da54dc5447559ab828865c332e1f7d14c Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Wed, 12 Mar 2025 15:27:07 +0100
Subject: [PATCH 820/954] feat(stm32mp2-fdts): remove ramp delay on ddr supply

Remove the unneeded regulator-enable-ramp-delay for DDR regulator vpp.
This timeout is no more required for board with STPMIC25 since the
default enable delay is set at 1ms for every regulator in PMIC driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 79d5bb6efb4b79c562b58ba9c208befaaa0c868a feat(stm32mp2-fdts): remove ramp delay on ddr supply]
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: Ia8e231d0a1086831fb4e56dd01d7ee7440072537
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/443762
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 fdts/stm32mp215f-dk.dts  | 1 -
 fdts/stm32mp235f-dk.dts  | 1 -
 fdts/stm32mp257f-dk.dts  | 1 -
 fdts/stm32mp257f-ev1.dts | 1 -
 4 files changed, 4 deletions(-)

diff --git a/fdts/stm32mp215f-dk.dts b/fdts/stm32mp215f-dk.dts
index f183a6d2a5..5e587bb509 100644
--- a/fdts/stm32mp215f-dk.dts
+++ b/fdts/stm32mp215f-dk.dts
@@ -111,7 +111,6 @@
 				regulator-name = "vdd1_ddr";
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
-				regulator-enable-ramp-delay = <1000>;
 			};
 			vdd3v3_usb: ldo4 {
 				regulator-name = "vdd3v3_usb";
diff --git a/fdts/stm32mp235f-dk.dts b/fdts/stm32mp235f-dk.dts
index 8953a575ad..7d39c8f3f6 100644
--- a/fdts/stm32mp235f-dk.dts
+++ b/fdts/stm32mp235f-dk.dts
@@ -132,7 +132,6 @@
 				regulator-name = "vdd1_ddr";
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
-				regulator-enable-ramp-delay = <1000>;
 			};
 			vdd3v3_usb: ldo4 {
 				regulator-name = "vdd3v3_usb";
diff --git a/fdts/stm32mp257f-dk.dts b/fdts/stm32mp257f-dk.dts
index 5eaa457b3b..5649d51997 100644
--- a/fdts/stm32mp257f-dk.dts
+++ b/fdts/stm32mp257f-dk.dts
@@ -132,7 +132,6 @@
 				regulator-name = "vdd1_ddr";
 				regulator-min-microvolt = <1800000>;
 				regulator-max-microvolt = <1800000>;
-				regulator-enable-ramp-delay = <1000>;
 			};
 			vdd3v3_usb: ldo4 {
 				regulator-name = "vdd3v3_usb";
diff --git a/fdts/stm32mp257f-ev1.dts b/fdts/stm32mp257f-ev1.dts
index b91c353bb8..848ab89925 100644
--- a/fdts/stm32mp257f-ev1.dts
+++ b/fdts/stm32mp257f-ev1.dts
@@ -143,7 +143,6 @@
 				regulator-name = "vpp_ddr";
 				regulator-min-microvolt = <2500000>;
 				regulator-max-microvolt = <2500000>;
-				regulator-enable-ramp-delay = <1000>;
 			};
 			vdd_sdcard: ldo7 {
 				regulator-name = "vdd_sdcard";
-- 
2.53.0

